services/frontend-service/src/ui/components/ReleaseDialog/ReleaseDialog.tsx

```diff
@@ -35,37 +35,52 @@ export const ReleaseDialog: React.FC<ReleaseDialogProps> = (props) => {
     const dialog =
```

Comment:
```suggestion
    const { version, app , ...etc} = props;
```
construct all the stuff form props then you don't have to write props.version, props.app everywhere